modules/ROOT/pages/dialog-components.adoc

```diff
@@ -446,6 +446,10 @@ A *listbox* is a composite component with a label and a dropdown list of options
 }
 ----
 
+[source,js]
```

Comment:
Is this addition intentional?

Reply:
Good catch. Looks like a code sample boilerplate inadvertently left in place. I’ll remove it.